evennia/commands/cmdsethandler.py

```diff
@@ -351,6 +351,14 @@ def update(self, init_mode=False):
                     elif path:
                         cmdset = self._import_cmdset(path)
                         if cmdset:
+                            if cmdset.key == '_CMDSET_ERROR':
+                                # If a cmdset fails to load, check if we have a fallback path to use
+                                fallback_path = settings.CMDSET_FALLBACKS.get(path, None)
```

Comment:
Alrighty, I added logging. I wasn't sure if it'd be appropriate to send some message to the object if `_IN_GAME_ERRORS` is set too. If that sounds like a good idea, I could add that in as well as doing the `logger.log_err` call.